src/lib/auth/github/permissions.js

```diff
@@ -5,7 +5,8 @@ const axios = require('axios')
 const config = require('../../../config')
 const logger = require('../../logger')
 
-const GITHUB_TEAMS_API_ENDPOINT = 'https://api.github.com/teams'
+const ALPHAGOV_GITHUB_ORGANISATION_ID = 596977
+const GITHUB_TEAMS_API_ENDPOINT = `/organizations/${ALPHAGOV_GITHUB_ORGANISATION_ID}/team`
```

Comment:
Don't you still need the domain part of the url?

Reply:
It’ll probably help, yes!

Reply:
I think the entire membership check url has changed https://docs.github.com/en/rest/reference/teams#get-team-membership-for-a-user